src/polyswarm_api/api.py

```diff
@@ -179,11 +175,11 @@ def rescan_id(self, scan, scan_config=None):
         :return: A ArtifactInstance resource
         """
         logger.info('Rescan id %s', int(scan))
-        return self.generator.rescanid(scan, scan_config=scan_config).execute().result
+        return resources.ArtifactInstance.rescanid(self, scan, scan_config=scan_config).result()
```

Comment:
I think this should be `rescan_id` to remain consistent with other names like `lookup_uuid`.

Reply:
:+1: